src/test/resources/pages/instructorCommentsPageSearchNormal.html

```diff
@@ -197,11 +197,11 @@
                     <b>
                       From:
                     </b>
-                    student1 In Course1\u003c/option\u003e\u003c/td\u003e\u003c/div\u003e\u0027\" (Team 1.1\u003c/td\u003e\u003c/div\u003e\u0027\")
```

Comment:
I don't really remember very clearly where these came from :P. @bamboo3250, do you recall?